spec/integration/estimates_spec.rb

```diff
@@ -86,7 +86,7 @@
       create_order: false
     )
 
-    expect(create_estimate_response.data.type).to eq 'shipping'
+    expect(create_estimate_response.data.type).to eq 'ecommerce'
```

Comment:
We're now re-routing to the ecommerce endpoint: https://github.com/patch-technology/patch/pull/5588